cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java

```diff
@@ -260,29 +251,15 @@ private String executeGroupAction(String currentUser, String actionName, int str
 		return this.executeAction();
 	}
 
-	private void checkReport(String commandId, int total, int applyTotal, int applySuccesses, int applyErrors, ApsCommandStatus status) {
-		BulkCommandReport<?> report = this._bulkCommandManager.getCommandReport(IContentBulkActionHelper.BULK_COMMAND_OWNER, commandId);
-		assertEquals(total, report.getTotal());
-		assertEquals(applyTotal, report.getApplyTotal());
-		assertEquals(applySuccesses, report.getApplySuccesses());
-		assertEquals(applyErrors, report.getApplyErrors());
-		assertEquals(status, report.getStatus());
-		if (total != applyTotal) {
-			assertNull(report.getEndingTime());
-		} else {
-			assertNotNull(report.getEndingTime());
-		}
-	}
-
-    @BeforeEach
+	@BeforeEach
 	private void init() {
 		this._contentManager = (IContentManager) this.getApplicationContext().getBean(JacmsSystemConstants.CONTENT_MANAGER);
-		this._bulkCommandManager = (IBulkCommandManager) this.getApplicationContext().getBean(SystemConstants.BULK_COMMAND_MANAGER);
+		//this._bulkCommandManager = (IBulkCommandManager) this.getApplicationContext().getBean(SystemConstants.BULK_COMMAND_MANAGER);
```

Comment:
Can we remove these commented lines?

Reply:
there was a comment in the original code in entando-ps, i will remove it

Reply:
done